client/my-sites/marketing/do-it-for-me/difm-lite-in-progress.tsx

```diff
@@ -83,39 +134,55 @@ function DIFMLiteInProgress( { siteId }: DIFMLiteInProgressProps ) {
 				}
 			) }
 			action={ translate( 'Manage domain' ) }
-			actionURL={ domainManagementList( slug ) }
+			actionURL={ domainManagementList( siteSlug ) }
 			secondaryAction={ hasEmailWithUs ? translate( 'Manage email' ) : translate( 'Add email' ) }
-			secondaryActionURL={ emailManagement( slug, null ) }
+			secondaryActionURL={ emailManagement( siteSlug, null ) }
 			secondaryActionCallback={ recordEmailClick }
 			illustration={ SiteBuildInProgressIllustration }
 			illustrationWidth={ 144 }
 			className="difm-lite-in-progress__content"
 		/>
-	) : (
-		<EmptyContent
-			title={ translate( 'Website content not submitted' ) }
-			line={ translate(
-				'Please provide the necessary information for the creation of your website. To access click on the button below.{{br}}{{/br}}' +
-					'{{SupportLink}}Contact support{{/SupportLink}} if you have any questions.',
-				{
-					components: {
-						br: <br />,
-						SupportLink: (
-							<a
-								href={ `mailto:builtby+express@wordpress.com?subject=${ encodeURIComponent(
-									`I need help with my site: ${ primaryDomain.domain }`
-								) }` }
-							/>
-						),
-					},
-				}
-			) }
-			action={ translate( 'Add content to your website' ) }
-			actionURL={ `/start/site-content-collection/website-content?siteSlug=${ slug }` }
-			secondaryActionCallback={ recordEmailClick }
-			illustration={ WebsiteContentRequiredIllustration }
-			illustrationWidth={ 144 }
-			className="difm-lite-in-progress__content"
+	);
+}
+
+function DIFMLiteInProgress( { siteId }: DIFMLiteInProgressProps ) {
+	const siteSlug = useSelector( ( state: AppState ) => getSiteSlug( state, siteId ) );
+	useQuerySitePurchases( siteId );
```

Comment:
I think it would be cleaner if we move this after the `const` defs. or does it need to sit here? 🤔